components/dashboard/src/settings/Integrations.tsx

```diff
@@ -248,6 +250,13 @@ function GitProviders() {
             />
         )}
 
+        {errorMessage && (
+            <div className="flex rounded-md bg-red-600 p-3">
```

Comment:
**thought**: This could be a good use case for introducing dismissible alerts but let's keep this out of the scope of this PR. We'll possibly work on dismissible alerts in https://github.com/gitpod-io/gitpod/issues/7613.